Examples/Io/Root/src/RootVertexPerformanceWriter.cpp

```diff
@@ -270,12 +445,29 @@ ActsExamples::ProcessCode ActsExamples::RootVertexPerformanceWriter::writeT(
             m_diffx.push_back(vtx.position()[0] - truePos[0]);
             m_diffy.push_back(vtx.position()[1] - truePos[1]);
             m_diffz.push_back(vtx.position()[2] - truePos[2]);
+
+            m_truthX.push_back(truePos[0]);
+            m_truthY.push_back(truePos[1]);
+            m_truthZ.push_back(truePos[2]);
+
+            m_recoX.push_back(vtx.position()[0]);
+            m_recoY.push_back(vtx.position()[1]);
+            m_recoZ.push_back(vtx.position()[2]);
+
+            m_covXX.push_back(vtx.covariance()(0, 0));
+            m_covYY.push_back(vtx.covariance()(1, 1));
+            m_covXY.push_back(vtx.covariance()(0, 1));
+            m_covYX.push_back(vtx.covariance()(1, 0));
+            m_trackVtxMatchFraction.push_back(trackVtxMatchFraction);
             // Next vertex now
             break;
           }
         }
       }
     }  // end loop vertices
+
+  } else {
+    throw std::invalid_argument{"Invalid truth input configuration"};
   }
```

Comment:
```suggestion
  }
```

if we remove the `else if` above